t/bulk.t

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use FindBin qw( $Bin );
+use lib qw( t/lib );
+use Test::More;
+use Sympa::Tools::File;
+use Sympa::Message;
+use ok 'Sympa::Bulk';
+use English;
+
+# during a test, syslog would be helpful as a note
+# so *Sympa::Log::syslog is diverted
+#
+# by default, syslog remains silent
+
+{ no warnings;
+    *Sympa::Log::syslog = 1 #  $ENV{SYMPA_TEST_SILENT_SYSLOG}
+    ? sub {}
+    : sub {
+        shift; # $self isn't expected anymore
+        my ( $level, $fmt, @params ) = @_;
+        note sprintf "syslog $level: $fmt", @params; }
+    ;
+}
+
+# this is the basic (ideally the only one keys required)
+# config to make this test work
+
+%Conf::Conf = (
+    # Sympa::Bulk::store needs
+    sympa_packet_priority => 5,
+    sympa_priority        => 1,
+    email                 => 'sympa',
+    domain                => 'lists.example.com',
+    sender_headers        => 'Resent-From,From,Return-Path',
+    list_check_suffixes   => 'list_check_suffixes',
+    umask                 => '027',
+);
+
```

Comment:
I can't understand why some of parameters above are assigned non-default values (especially, value for list_check_suffixes is obviously illegal).

Reply:
list_check_suffixes is clearly a typo! thanks for spotting it. the others are stolen from the ConfDefs.pm AFAIR. do you want me to check all of them to be the default ones (assuming defaults comes from ConfDefs.pm) ?

Reply:
Test would be performed under controlled conditions. Otherwise, it might be hard to distinct unexpected result by a bug and unexpected result by randomly chosen condition.  Non-specific value is prefered for parameters unless there were reason to adopt special value.